cmd/dao_tests/pubkey_test.go

```diff
@@ -10,19 +10,30 @@ import (
 
 	"github.com/RHEnVision/provisioning-backend/internal/dao"
 	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/RHEnVision/provisioning-backend/internal/config"
 	"github.com/RHEnVision/provisioning-backend/internal/db"
 )
 
+func createPk() []*models.Pubkey {
```

Comment:
Why are we returning an array and then dereferencing it in every use?
Wouldn't it be easire to just return the pubkey?